cypress/e2e/v1.0/pages/offence-analysis/questions/victim/victimRaceOrEthnicity.ts

```diff
@@ -0,0 +1,94 @@
+const question = "What is the victim's race or ethnicity?"
+const options = [
+  'White - English, Welsh, Scottish, Northern Irish or British',
+  'White - Irish',
+  'White - Gypsy or Irish Traveller',
+  'White - Any other White background',
+  'Mixed - White and Black Caribbean',
+  'Mixed - White and Black African',
+  'Mixed - White and Asian',
+  'Mixed - Any other mixed or multiple ethnic background background',
+  'Asian or Asian British - Indian',
+  'Asian or Asian British - Pakistani',
+  'Asian or Asian British - Bangladeshi',
+  'Asian or Asian British - Chinese',
+  'Asian or Asian British - Any other Asian background',
+  'Black or Black British - Caribbean',
+  'Black or Black British - African',
+  'Black or Black British - Any other Black background',
+  'Any other ethnic group',
+  'Not stated',
+]
```

Comment:
I wonder if a reference data file would have been useful in this project, so that the app/tests could get these arrays from a single place.

Reply:
Totally agree, it might be worth doing a refactor of the tests in the future to externalise the content, would certainly be easier to maintain

Reply:
Had a good conversation on this, one to maybe implement in future 😄